superdesk/io/feeding_services/file_service.py

```diff
@@ -61,7 +61,7 @@ def _update(self, provider):
                             with open(file_path, 'rt') as f:
                                 xml = ElementTree.parse(f)
                                 parser = self.get_feed_parser(provider, xml.getroot())
-                                item = parser.parse(xml, provider)
+                                item = parser.parse(xml.getroot(), provider)
```

Comment:
We don't currently have any file/xml ingests configured. I have tested a Reuters with a file feeding service and it requires this fix, haven't tested nitf source.